enabler/src/de/schildbach/pte/AbstractNavitiaProvider.java

```diff
@@ -862,24 +862,35 @@ public QueryDeparturesResult queryDepartures(final String stationId, final @Null
 			// to stop_point and query departures.
 			final StringBuilder queryUri = new StringBuilder();
 			queryUri.append(uri());
-			if (equivs)
+			final String header = stationId.substring(0, stationId.indexOf(":"));
```

Comment:
That is very possible. There's a bunch of other problems with this provider as well that I am currently looking into.
Please note that I didn't add this line in this PR, but merely moved it.